src/ms.tex

```diff
@@ -287,9 +287,60 @@ \subsection{Funding} \label{sec:project-funding}
 
 \secauthor{Aarya Patil}
 
-Summarize funding sources (Moore, NASA) and amounts and what this has been used
-for.
+% Summarize funding sources (Moore, NASA) and amounts and what this has been
+% used for.
+
+With the goal of establishing a sustainable financial model for the Astropy
+project, the Coordination Committee under the new governance charter (see \S
+\ref{sec:project-governance}) established a standing Finance committee.
+This committee overlooks the planning and disbursement of finances on behalf
```

Comment:
```suggestion
This committee oversee planning and allocation of finances on behalf
```
maybe it's just me, but "disbursement" seems a bit less community-centric, and this sounds better to me